### ISSUE TITLE
Cut a new release

### ISSUE THREAD

**Issue by kevinsung** [quantumlib/OpenFermion-Psi4#35]
#34 is a pretty crucial change; without it installation fails for some systems.

**Comment by loriab** [comment]
I don't think it got a tag here on GH, but looks like the 0.4 release did appear on PyPI.

FYI, just released v1.3 of Psi4. Both openfermion pacakges are available through conda for Mac/Linux py36/37. Both are included in the `psi4-rt` metapackage of optional run-time dependencies. (click "conda" to see contents in figure below at https://admiring-tesla-08529a.netlify.com/installs/v13/ .) that page will move to psicode.org in the next few days.

* https://anaconda.org/psi4/openfermionpsi4
* https://anaconda.org/psi4/openfermion

**Comment by dgasmith** [comment]
Should probably make a separate issue about how to integrate the new python-based T1/T2 amplitude read so that the ASCII text parsing can be deprecated here.
